plugins/woocommerce/client/blocks/assets/js/base/components/cart-checkout/place-order-button/index.tsx

```diff
@@ -41,9 +47,32 @@ const PlaceOrderButton = ( {
 		waitingForRedirect,
 	} = useCheckoutSubmit();
 
+	const paymentMethodInterface = usePaymentMethodInterface();
+	const { isEditor, isPreview = false } = useEditorContext();
```

Comment:
Here I wasn't sure whether to use a separate attribute on the main `registerExpressPaymentMethod` config (like `edit`/`content`). I opted for different props 🤷

Reply:
I think this is fine, tbh asking for different buttons for edit/content feels like it would just add an extra layer of friction to the integration process.